src/constants.js

```diff
@@ -541,6 +541,8 @@ export const SHOW_UNTAGGED_FILTER = 'showUntagged'
 export const SORT_BY = 'sortBy'
 export const STATUS_FILTER = 'status'
 export const TAG_FILTER = 'tag'
+export const REFRESH = 'refresh'
```

Comment:
rename to AUTO_REFRESH_ID = 'auto-refresh'

Reply:
@mavdryk done.